incubator/java-openliberty/README.md

```diff
@@ -118,6 +118,34 @@ The metrics endpoint is secured with a userid and password enabled through the c
 
 In order to lock down the production image built via `appsody build` this file is deleted during the Docker build of your application production image.  (The same file would be deleted if you happened to create your own file at this location as well).
 
+## Stack development
+
+The Java Open Liberty stack is fully functional out of the box. However, depending on business needs and requirements, it may become necessary to create a custom stack with additional features and enhancements. This can be done by following the steps outlines here: https://appsody.dev/docs/stacks/develop
+
+Additionally, the Java Open Liberty stack includes a set of custom variables and enforcement rules that can be leveraged when modifying the stack.
+
+### Stack template variables
+
+Custom template variables are defined in the `stack.yaml` file. The Java Open Liberty stack includes the following that are propagated when the stack is packaged.
+
+* **libertyversion** - The version of the Open Liberty runtime to be used for development and production images. Without any other modifications to the Dockerfiles in this project, this version must be a quarterly release (x.0.0.3, x.0.0.6, x.0.0.9, x.0.0.12)
+
+* **parentpomgroup** - The group id used for the parent pom definition
```

Comment:
.  We also use this to form the "prefix" of the groupid used by the default template artifact.